lib/drivers/abbottFreeStyle.js

```diff
@@ -373,8 +373,9 @@ module.exports = function (config) {
       // once we've sent the command, start listening for a response
       // but if we don't get one before the end of the timeout, give up
       // max timeout is 1000 * 32 bytes / 19200 baud / 10 bits/char == about 16 sec
```

Comment:
Does your change match the comment here still? If it doesn't, can you update the comment?